boot/freeldr/freeldr/arch/i386/machxbox.c

```diff
@@ -23,6 +23,53 @@
 DBG_DEFAULT_CHANNEL(HWDETECT);
 
 
+extern
+VOID
+DetectSerialPointerPeripheral(PCONFIGURATION_COMPONENT_DATA ControllerKey, PUCHAR Base);
+
+static
+ULONG
+XboxGetSerialPort(ULONG Index, PULONG Irq)
+{
+    /*
+     * Xbox may have maximum two Serial COM ports
+     * if the Super I/O chip is connected via LPC
+     */
+    UCHAR Device[MAX_XBOX_COM_PORTS] = {LPC_DEVICE_SERIAL_PORT_1, LPC_DEVICE_SERIAL_PORT_2};
```

Comment:
static variable (looks like some kind of constant data to me; same remark as above).